entrypoint.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-{ echo "PACKER_VERSION=1.7.3"; echo "TERRAFORM_VERSION=0.12.31"; echo "TERRAFORM_DOCS_VERSION=v0.14.1"; } >> "$GITHUB_ENV"
+{ echo "PACKER_VERSION=1.7.3"; echo "SHFMT_VERSION=v3.3.0"; echo "TERRAFORM_VERSION=0.12.31"; echo "TERRAFORM_DOCS_VERSION=v0.14.1"; } >> "$GITHUB_ENV"
```

Comment:
This line is getting pretty long- can we get a line break in there?

Reply:
My thinking was that we'd see what the resulting Lineage PR that will flow down from cisagov/skeleton-generic#84 does before intervening manually.  Part of the reason for these linters and formatters is to avoid things of that ilk.

Reply:
That makes sense. 👍